File: VB/SilverlightApplication15/PageViewModel.Partial.vb
Imports System.ComponentModel
Imports DevExpress.Xpf.Printing

Partial Public Class PageViewModel
#Region "INotifyPropertyChanged"
    Public Event PropertyChanged As PropertyChangedEventHandler Implements INotifyPropertyChanged.PropertyChanged

    Sub RaisePropertyChanged(ByVal propertyName As String)
        RaiseEvent PropertyChanged(Me, New PropertyChangedEventArgs(propertyName))
    End Sub
#End Region
End Class

File: CS/SilverlightApplication15/PageViewModel.Partial.cs
using System.ComponentModel;

namespace SilverlightApplication15 {
    public partial class PageViewModel {
        #region INotifyPropertyChanged
        public event PropertyChangedEventHandler PropertyChanged;

        void RaisePropertyChanged(string propertyName) {
            if(PropertyChanged != null) {
                PropertyChanged(this, new PropertyChangedEventArgs(propertyName));
            }
        }
        #endregion
    }
}
